Doc/includes/email-alternative.py

```diff
@@ -8,14 +8,14 @@
 
 # Create the base text message.
 msg = EmailMessage()
-msg['Subject'] = "Ayons asperges pour le déjeuner"
+msg['Subject'] = "Pourquoi pas des asperges pour ce midi?"
```

Comment:
The problem with this particular fix is that part of the point of the test is to have a non-ascii character in it, and this fixed version does not.  Choose a whole new sentence if you want to, but it must contain a non-ascii character.